hkm/templatetags/hkm_tags.py

```diff
@@ -19,18 +19,21 @@ def finna_image(img_id):
 
 @register.filter
 def truncate_description(value):
-    result = trim_prefix(value, ["sisällön kuvaus: "])
-    return trim_suffix(result, ["mustavalkoinen", "värillinen"])
+    result = value.removeprefix("sisällön kuvaus: ")
+    result = result.removesuffix("mustavalkoinen")
+    result = result.removesuffix("värillinen") 
+    return result
 
 
 @register.filter
 def truncate_era(value):
-    return trim_prefix(value, ["kuvausaika: "])
+    return value.removeprefix("kuvausaika ")
```

Comment:
Did you change the removable prefix "kuvausaika: " to "kuvausaika " on purpose here?